prac_07/myguitars.py

```diff
@@ -16,10 +16,22 @@ def main():
     guitars = load_guitars_from_csv(FILENAME)
     guitars.sort()
     print("Guitars List.")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitars.append(Guitar(name, year, cost))
+        print(f"{name} ({year}) : ${cost:.2f} added.")
```

Comment:
Since you have the __str__ method with the formatting already specified in your Guitar class, would it follow the DRY principle to print the object next to "added"?